src/core/abstractions/useSampler/index.ts

```diff
@@ -0,0 +1,142 @@
+import { ref, watchEffect } from 'vue'
+import {
+  Vector3,
+  Color,
+  Object3D,
+  InterleavedBuffer,
+} from 'three'
+import type { Mesh, InstancedMesh } from 'three'
+import { MeshSurfaceSampler } from 'three/examples/jsm/math/MeshSurfaceSampler'
```

Comment:
```suggestion
import { MeshSurfaceSampler } from 'three-stdlib'
```

Avoid exporting these extra classes from three since they are not three-shakeable and not ESM compatible, use `three-stdlib` instead